parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -98,6 +105,12 @@ installCrictl() {
 downloadTeleportdPlugin() {
     DOWNLOAD_URL=$1
     TELEPORTD_VERSION=$2
+    CPU_ARCH=$(getCPUArch)  #amd64 or arm64
```

Comment:
hmm I see the logic used atleast in 4 places. thoughts on moving it to a helper function?

Reply:
thanks for the comments. I added a new function isARM64() for this purpose.